brave-spancollector-kafka/README.md

```diff
@@ -0,0 +1,4 @@
+# brave-spancollector-kafka #
+
+SpanCollector that is used to submit spans to Kafka.
+It submits spans to a topic named `zipkin`.
```

Comment:
Spans are sent to kafka as keyed messages: the key is the topic `zipkin` and the value is a TBinaryProtocol encoded Span.